### ISSUE TITLE
Plugin breaks context "Show Context Actions"

### ISSUE THREAD

**Issue by maxanier** [Rohde-Schwarz/RcIcPluginBugTracking#5]
Hello,

when I activate this Plugin, the "Show Context Action" menu in Pycharm is not working anymore (neither from right-click menu nor by Alt+Enter).
The Yellow Light-bulb still shows up for inspections and allows executing some context actions, only the general context action menu does not work at all.
If I disable the plugin, everything works again.
This affects Python files, but also other languages for me.

**With Plugin**
![broken](https://github.com/user-attachments/assets/152668e8-6603-4b23-acf8-5fd2f32df002)
**Without Plugin**
![normal](https://github.com/user-attachments/assets/3c20a9a8-4ac9-4b71-b30c-dd4e50c645f2)

PyCharm CE: 2024.2
Rohde & Schwarz Instrument Connectivity: 3.2.0 **Also tested with 3.4.0**
Windows 10

Can you reproduce this issue, or should I provide further log files?

Many Thanks
Max

**Comment by maxanier** [comment]
Also happens with version 3.4.0 of this plugin

**Comment by Miloslav-RS** [comment]
Oh man, sorry, Max, I overlooked your post completely. I tested it now with the plugin 3.10.0 and Pycharm 2025.1, could not reproduce the issue. Which file type did you test? I tried the .py file, but the line is obviously not the python syntax.

Milo

**Comment by maxanier** [comment]
I have created a new project with plugin 3.10.0 and Pycharm 2025.1.1.1 CE on Windows with all other plugins disabled.
The problem persists :/

```
#test.py
print("Hello")

4 + 5

print("Goodbye")
```

With this plugin disabled, the context action `introduce variable for statement` is offered for the useless statement in the middle.
With the plugin enabled, it says "No context actions available at this location".
The `Statement seems to have no effect` warning and yellow highlighting is shown regardless

The `idea.log` file does not contain anything that seems interesting unfortunately.

**Comment by Miloslav-RS** [comment]
You're right, I can reproduce it with 3.10.0. Let me check where it comes from.
That means the context-menu works now (meaning, it appears), just no context-actions are presented? This is how it behaves on my end.

I have a couple of issues in the backlog to create a 3.10.1 bugfix version. Anyway, thanks for reporting this and for the patience :-)

Milo

**Comment by maxanier** [comment]
Perfect. Yes, the menu appears, but no context actions are present.

Thank you for looking into this :)

Max

**Comment by Miloslav-RS** [comment]
Found the issue, it was a relic from an older version - it affects and blocks all the intentions in non-SCPI files. Will be fixed in the version 3.10.1 coming out next week.

Milo
